src/embed/init.ts

```diff
@@ -79,7 +91,7 @@ function _dlo_collectRules(): any[] {
   }
 }
 
-function _dlo_initializeFromWindow() {
+export default function _dlo_initializeFromWindow() {
```

Comment:
I'm not thrilled about exporting this function to facilitate testing, but this seemed like a narrow and low risk way to enable testing of telemetry initialization from window configuration. Note: This change results in a new warning when building DLO:

> (!) If you do not supply "output.name", you may not be able to access the exports of an IIFE bundle.

This export is only used from tests and does not seem to cause any testing issues.

Reply:
Not a huge deal - but it might be cool if there were a way to suppress that warning or document what's causing it and why it's safe to be ignored.

Reply:
I moved initialization functions to a separate file where they could be exported for both library initialization and simulated initialization from tests without also being coupled to IIFE. This addressed the rollup build warning.